### PR TITLE
Some bee debug shit

### PR DESCRIPTION
**Pull request by DeityLink**
Adds a new button to the Debug tab, requiring the +DEBUG flag, that gives its user the current count of existing bee datums, existing bee swarms, and sum of all bees contained within apiaries.

So the next time nuke lags the server with bees, just press that button and send me the data.

### CODE REVIEW

**Comment by Kharkov72** [comment]
🐝

**Comment by nuklearcellphoneg** [comment]
🐝

**Comment by ghost** [comment]
🅱️

**Comment by spoonylove** [comment]
This should be a thing for the end of round score and should tell the players if botany broke a new high score with bees

**Comment by nuklearcellphoneg** [comment]
Don't tempt me

**Comment by Killette2** [comment]
Alright nuke playtime is over, we got a scoreboard now.

**Comment by DeityLink** [comment]
I don't know what travis' problem is, there's no error.

**Comment by Exxion** [comment]
I can already tell you that unless the ungodly hell-lag we thought had been fixed has come back, the lag is caused by reagent transfers to apiaries

File: code/modules/mob/living/simple_animal/bees/bees_datums.dm [hunk 1]
```diff
@@ -54,6 +57,7 @@
 	qdel(src)
 
 /datum/bee/Destroy()
+	bees_count--
```
Comment (Exxion) [inline code/modules/mob/living/simple_animal/bees/bees_datums.dm:62]: This should be in `Del()`, not `Destroy()`
It shouldn't really matter but is more foolproof that way

**Comment by nuklearcellphoneg** [comment]
@Exxion So if I raised plants on plants with no reagents in them, do you think they wouldn't lag? Or would production of honey prolly still cause that?

**Comment by Exxion** [comment]
Fewer reagents should lag less, I think? I wouldn't say it wouldn't lag at all, though

**Comment by DeityLink** [comment]
The most simple way would be to check the profiler on the test server.

**Comment by DeityLink** [comment]
Alright conflicts fixed

**Pull request merged**